netlify.toml

```diff
@@ -157,40 +157,48 @@
 [[redirects]]
     from = "/docs/plugins/types"
     to = "/docs/plugins/build/types"
-    
+
 [[redirects]]
     from = "/docs/developing-locally"
     to = "/docs/contributing/developing-locally"
 
-    
+
 [[redirects]]
     from = "/docs/project-structure"
     to = "/docs/contributing/project-structure"
 
-    
+
 [[redirects]]
     from = "/docs/recognizing-contributions"
     to = "/docs/contributing/recognizing-contributions"
 
-    
+
 [[redirects]]
     from = "/docs/stack"
     to = "/docs/contributing/stack"
 
-    
+
 [[redirects]]
     from = "/docs/updating-documentation"
     to = "/docs/contributing/updating-documentation"
 
+[[redirects]]
+    from = "/docs/self-host"
+    to = "/docs/self-host/overview"
+
 [[redirects]]
     from = "/docs/deployment"
-    to = "/docs/self-hosting/overview#deploy"
+    to = "/docs/self-host/overview"
```

Comment:
```suggestion
    to = "/docs/self-host/overview#deploy"
```

Reply:
🤦  (by the time I found the right place to change I had already my mind set on redirecting to just overview).